TabNine.py

```diff
@@ -50,6 +51,7 @@ def join_path(*args):
 
 class TabNineProcess:
     install_directory = os.path.dirname(os.path.realpath(__file__))
+    pack_manager = package_manager.PackageManager()
```

Comment:
Is there a reason this should be a part of the class?

Reply:
Only requirement from this var is to be accessible in the `run_tabnine` static method.

Reply:
I would just make it a global variable in the file